src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -244,6 +244,28 @@
             },
             'required': ['type', 'title', 'elements'],
         },
+
+        'stacktrace-link': {
+            'type': 'object',
+            'properties': {
+                'type': {
+                    'type': 'string',
+                    'enum': ['stacktrace-link'],
+                },
+                'uri': {
+                    '$ref': '#/definitions/uri',
+                },
+                'params': {
```

Comment:
If we're scoping to a line in a stacktrace, I don't know that we need to make `params` configurable any longer. I don't know what other than a filename and line number you can derive from a line of code.